src/it/junit/custom-output-directory/verify.groovy

```diff
@@ -15,8 +18,8 @@ import cucumber.api.CucumberOptions;
 import cucumber.api.junit.Cucumber;
 
 @RunWith(Cucumber.class)
-@CucumberOptions(strict = true, features = {"classpath:features/feature1.feature"}, plugin = {"json:target/my-custom-dir/1.json",
-"pretty"}, monochrome = false, tags = {"@complete", "@accepted"}, glue = { "foo", "bar" })
+@CucumberOptions(strict = true, features = {\"""" + feature1.absolutePath + """\"}, plugin = {"json:target/my-custom-dir/1.json"},
```

Comment:
Instead of having to split the """,

You should be able to use:
`...features = {" ${feature1.absolutePath}"}...`

I don't think the `"` quotes should be escaped - doesn't that make it into the final file and result in a compilation error?